.circleci/create_circleci_config.py

```diff
@@ -264,6 +248,25 @@ def to_dict(self):
 
         steps.append({"store_artifacts": {"path": "~/transformers/tests_output.txt"}})
         steps.append({"store_artifacts": {"path": "~/transformers/reports"}})
+
+        # save cache at the end: so pytest step runs before cache saving and we can see results earlier
```

Comment:
see them earlier on the console. The final results will be shown on a tab only if the whole job is finished.